about.css

```diff
@@ -0,0 +1,510 @@
+    background: #272a31;
+    padding: 10px 15vw;
+    color: white;
+    font-size: 10px;
+  }
+
+  .header .desktop-nav .white-row {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background: white;
+    padding: 15px 15vw;
+  }
+
+  .header .desktop-nav .white-row .list {
+    display: flex;
+    gap: 20px;
+  }
+
+  .header .desktop-nav .white-row .a {
+    color: #9d9d9d;
+    font-size: 15px;
+  }
+
+  .header .desktop-nav .white-row .a:hover {
+    color: #ec5242;
+  }
+
+  .header .desktop-nav .white-row .a-cc {
+    border: solid 4px red;
+    padding: 10px;
+    color: #ec5242;
+    white-space: nowrap;
+  }
+
+  /* Section 1 */
+
+  .section-1 {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 20px;
+    margin: 0;
+    padding: 150px 20vw 150px 20vw;
+    text-align: center;
+    background-image: url('images/home-bg-desktop.png');
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  .section-1 .hello {
+    display: none;
+  }
+
+  .section-1 .h1 {
+    color: #ec5242;
+    font-size: 50px;
+    line-height: 50px;
+    margin-bottom: 0;
+    max-width: 460px;
+  }
+
+  .section-1 .red-line {
+    height: 1px;
+    width: 25px;
+    background: #ec5242;
+  }
+
+  .section-1 .p {
+    background: white;
+    border: 1px solid #e3e2e2;
+    padding: 20px;
+    line-height: 25px;
+    font-size: 14px;
+    color: #7a7a7a;
+    max-width: 715px;
+  }
+
+  .section-1 .mobile-p {
+    display: none;
+  }
+
+  .section-1 .desktop-p {
+    display: block;
+  }
+
+  .section-1 .email {
+    color: #5f5f5f;
+    text-decoration: underline;
+    text-underline-position: under;
+    padding-bottom: 5px;
+    font-size: larger;
+    font-weight: 500;
+    margin-top: -10px;
+  }
+
+  /* Section 2 */
+
+  .section-2 {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 50px;
+    padding: 50px 20vw;
+    text-align: center;
+  }
+
+  .section-2 .h2 .br {
+    display: none;
+  }
+
+  .section-2 .mobile-p {
+    display: none;
+  }
+
+  .section-2 .desktop-p {
+    display: block;
+    max-width: 580px;
+    font-size: 12px;
+    line-height: 20px;
+  }
+
+  .section-2 .event-container {
+    display: flex;
+    flex-direction: row;
+    gap: 20px;
+  }
+
+  .section-2 .event-container .event {
+    height: 300px;
+    width: 35vw;
+  }
+
+  .section-2 .event .p {
+    font-size: 20px;
+  }
+
+  .section-4 {
+    display: none;
+  }
+
+  /* Footer */
+
+  .footer {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    gap: 5vw;
+    padding: 50px 15vw;
+    background-color: #2e2e2e;
+  }
+
+  .footer .logo {
+    height: 40px;
+    width: 210px;
+  }
+
+  .footer .text {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+    max-width: 550px;
+    font-size: 14px;
+    font-weight: 700;
+    color: #fff;
+  }
```

Comment:
- [ ] Well done for changing the background color for mobile version's footer 🥇 However, please note that the previous reviewer didn't suggest to change the `color` of the text and logo for desktop version. I think that happened while implementing the change for mobile version😅 The footer is looking a bit different now (desktop version). Can you  please revert back the color change for desktop version? 🤓🔨

Your design before | Your design now 
---| ---|
![image](https://github.com/suthish777/capstone__01/assets/106119105/339d8dc1-32a3-40a4-929b-66d70cced482) | ![image](https://github.com/suthish777/capstone__01/assets/106119105/61693732-eb54-4e55-b30c-7dc25e0a4eb0)